o.apache.xml.resolver/external/xml-resolver-1.2-netbeans.patch

```diff
@@ -0,0 +1,822 @@
+diff -ruN org/apache/xml/resolver/Catalog.java patched-org/apache/xml/resolver/Catalog.java
```

Comment:
I think adding an apache header to this patch would be a good idea, so if someone looks into this, the situation is clear.